include/fluent-bit/flb_str.h

```diff
@@ -58,4 +58,13 @@ static inline char *flb_strndup(const char *s, size_t n)
     return str;
 }
 
+/* emptyval checks whether a string has a non-null value "". */
+static inline int flb_str_emptyval(const char *s)
```

Comment:
this function is only called from flb_config.c file, let's move it there in the .c file on top as a static inline.

Reply:
DONE.